fixtures/worker-app/worker-configuration.d.ts

```diff
@@ -1,5 +1,5 @@
-// Generated by Wrangler on Thu Mar 28 2024 11:27:36 GMT+0000 (Greenwich Mean Time)
-// by running `wrangler types`
+// Generated by Wrangler on Mon Jul 22 2024 10:41:44 GMT+0200 (Central European Summer Time)
+// by running `wrangler types --x-with-runtime`
```

Comment:
Would be great if this file was reproducible without a diff each time `wrangler types --with-runtime` was run.

Is the timestamp necessary? I think @penalosa's idea of using the workerd version and compat date+flags would work better

Reply:
Looking back through the history, and the [original PR](https://github.com/cloudflare/workers-sdk/pull/2074/files#diff-3ffb2d8628f77f1a82a17c8d3d13371329bd2a1846cff95da89c41acfa5a3edfR152) that introduced this, there doesn't seem to be a specific reason to keep it.

Reply:
@RamIdeas @penalosa can I suggest doing this in a follow-up PR? It requires some refactoring so might be better of as an atomic set of changes.